vss-manifest.js

```diff
@@ -96,6 +96,26 @@ module.exports = () => {
                     dataType: 'string',
                   },
                 },
+                {
+                  id: 'aquaUrl',
+                  name: 'Aqua Platform URL',
+                  inputMode: 'textbox',
+                  isConfidential: false,
+                  validation: {
+                    isRequired: true,
```

Comment:
I suppose, this should be not required. 
If it's required and user should set it as described in `connectedservice.md` , then it's better to use top `url` property for this.

Reply:
Yeah, these are required, I'll move them